index.html

```diff
@@ -1,282 +1,291 @@
+                fill="#1C1C1C"
+              ></path>
+            </svg>
+            KyberAI
+            <div
+              style="
+                padding: 2px 4px;
+                color: rgb(169, 169, 169);
+                background-color: rgba(169, 169, 169, 0.196);
+                border-radius: 12px;
+                font-size: 10px;
+                margin-top: -16px;
+              "
+            >
+              beta
+            </div>
+          </div>
+
+          <div class="preloadhtml-header-link-item">Campaigns</div>
+          <div class="preloadhtml-header-link-item">Analytics</div>
+          <div class="preloadhtml-header-link-item">About</div>
+          <div class="preloadhtml-header-link-item">Blog</div>
+        </div>
+      </div>
+      <div style="height: 180px; display: flex">
+        <img
+          class="onlyVisibleInDarkMode"
+          style="animation: pulse 800ms linear infinite; margin: auto"
+          width="160px"
+          src="/logo-dark.svg"
+          alt="DmmLogo"
+        />
+        <img
+          class="onlyVisibleInLightMode"
+          style="animation: pulse 800ms linear infinite; margin: auto"
+          width="160px"
+          src="/logo.svg"
+          alt="DmmLogo"
+        />
+      </div>
+    </div>
+    <script>
+      ;(() => {
+        function doesUseDarkMode() {
+          const rawLocalState = window.localStorage.getItem('redux_localstorage_simple_user')
+          const hasRawLocalState = typeof rawLocalState === 'string'
+          if (hasRawLocalState) {
+            try {
+              // need JSON.parse here as we useLocalStorage when storing theme mode
+              const localState = JSON.parse(rawLocalState)
+              return localState.userDarkMode
+            } catch (e) {
+              // empty
+            }
+          }
+        }
+
+        function setInitialColorMode(usePrefersColorScheme) {
+          function getInitialColorMode() {
+            const useDarkMode = doesUseDarkMode()
+            if (typeof useDarkMode === 'boolean') {
+              return useDarkMode ? 'dark' : 'light'
+            }
+
+            if (usePrefersColorScheme === false) {
+              // default to 'dark'.
+              return 'dark'
+            }
+
+            // If there is no saved preference, use a media query
+            const mediaQuery = '(prefers-color-scheme: dark)'
+            const mql = window.matchMedia(mediaQuery)
+            const hasImplicitPreference = typeof mql.matches === 'boolean'
+            if (hasImplicitPreference) {
+              return mql.matches ? 'dark' : 'light'
+            }
+
+            // default to 'dark'.
+            return 'dark'
+          }
+          const colorMode = getInitialColorMode()
+          // add HTML attribute if dark mode
+          if (colorMode === 'dark') {
+            document.documentElement.setAttribute('data-theme', 'dark')
+          }

```

Comment:
This is missing. Please add it again
https://github.com/KyberNetwork/kyberswap-interface/blob/main/index.html#L248